media_utils/video_puppeteer.py

```diff
@@ -56,12 +39,13 @@ def __init__(self, marionette, url, video_selector='video'):
                                                           'Using first.')
             if len(videos_found) > 0:
                 self.video = videos_found[0]
-                self.marionette.execute_script("log('video element obtained');")
-
-    def get_debug_lines(self):
```

Comment:
SydPolkzillaMBP:firefox-media-tests spolk$ find . -type f -exec grep -H get_debug_lines '{}' \;
SydPolkzillaMBP:firefox-media-tests spolk$ git checkout -b mjzffr-master mjzffr/master
Branch mjzffr-master set up to track remote branch master from mjzffr.
Switched to a new branch 'mjzffr-master'
SydPolkzillaMBP:firefox-media-tests spolk$ find . -type f -exec grep -H get_debug_lines '{}' \;
./media_utils/video_puppeteer.py:    def get_debug_lines(self):
./media_utils/video_puppeteer.py:    debug_lines = video.get_debug_lines()
SydPolkzillaMBP:firefox-media-tests spolk$ 

The instance I see there was the old code. I don't see where it is called in testcase.py.